rfd/00XX-ci.md

```diff
@@ -0,0 +1,83 @@
+
+---
+author: jane quintero (jane@goteleport.com) Russell Jones (rjones@goteleport.com)
+state: draft
+---
+
+# Continuous Integration (CI)
```

Comment:
nit: the RFD name is misleading

CI usually just means "a thing that runs tests" and does not include other automation around PRs (assignees, approvals).
Can this be renamed to `Development workflow automation`, with CI being a subset of this?